test/api_tests.coffee

```diff
@@ -74,28 +74,19 @@ Meteor.startup ->
 
 
   describe 'A collection route', ->
-    it 'should be able to exclude endpoints using just the excludedEndpoints option', (test, next) ->
+    it 'should be able to exclude endpoints using just the excludedEndpoints option', (test) ->
       Restivus.addCollection new Mongo.Collection('tests2'),
         excludedEndpoints: ['get', 'getAll']
 
+      # Since the endpoints don't exist, they just return the Meteor HTML code
+      result = HTTP.get 'http://localhost:3000/api/v1/tests2/10'
+      test.isTrue /__meteor_runtime_config__/.test(result.content)
 
-      HTTP.get 'http://localhost:3000/api/v1/tests2/10', (error, result) ->
-        response = JSON.parse result.content
-        test.isTrue error
-        test.equal result.statusCode, 404
-        test.equal response.status, 'error'
-        test.equal response.message, 'API endpoint not found'
-
-      HTTP.get 'http://localhost:3000/api/v1/tests2/', (error, result) ->
-        response = JSON.parse result.content
-        test.isTrue error
-        test.equal result.statusCode, 404
-        test.equal response.status, 'error'
-        test.equal response.message, 'API endpoint not found'
-        next()
+      result = HTTP.get 'http://localhost:3000/api/v1/tests2/'
+      test.isTrue /__meteor_runtime_config__/.test(result.content)
 
     context 'with the default autogenerated endpoints', ->
-      Restivus.addCollection new Mongo.Collection('testautogen')
+      Restivus.addCollection new Mongo.Collection('testAutogen')
```

Comment:
This is what I meant by maybe accidentally making endpoints case-sensitive. I think this conforms to standards, but might be an unwanted API change.